### PR TITLE
Make warnings

### PR DESCRIPTION
**Pull request by glebdronchenko**

### CODE REVIEW

File: src/components/Warnings/Warnings.jsx [hunk 1]
```diff
@@ -1,12 +1,33 @@
 import classes from './Warnings.module.scss';
 import Warning from './Warning/Warning';
+import EmptyListMessage from '../EmptyListMessage/EmptyListMessage';
+import { useCookies } from 'react-cookie';
```
Comment (anton-vlasik) [inline src/components/Warnings/Warnings.jsx]: ?

File: src/components/Warnings/Warnings.jsx [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+import ApiService from '../../services/ForecastApiService';
+import mockData from './mockData';
+
+const warningsList = warnings => warnings.map(warning => <Warning data={warning} />);
 
 function Warnings() {
+  const [warnings, setWarnings] = useState([]);
+
+  useEffect(async () => {
+    const { data } = await ApiService.getWarnings('105128581');
```
Comment (anton-vlasik) [inline src/components/Warnings/Warnings.jsx]: '105128581' - ? again?

File: src/components/Warnings/Warnings.jsx [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+    setWarnings(data.warnings);
+  }, []);
+
   return (
     <div className={classes.warningsContainer}>
       <h2 className={classes.title}>Warnings</h2>
       <div>
-        <Warning />
+        {warnings.length ? (
+          warningsList(warnings)
```
Comment (anton-vlasik) [inline src/components/Warnings/Warnings.jsx]: This is an inappropriate name for a function.

File: src/components/Warnings/Warnings.jsx [hunk 4]
```diff
@@ -1,12 +1,33 @@
...
+        ) : (
+          <EmptyListMessage
+            mainMessage={`Warnings list is empy`}
+            additionalMessage={`Have a nice day!`}
```
Comment (anton-vlasik) [inline src/components/Warnings/Warnings.jsx]: What would be the next name if there will be another line of words in your EmptyListMessage? 
superAdditional message?
tinyMessageAtTheBottom?

File: src/components/Warnings/Warnings.jsx [hunk 5]
```diff
@@ -1,12 +1,33 @@
...
   return (
     <div className={classes.warningsContainer}>
       <h2 className={classes.title}>Warnings</h2>
       <div>
-        <Warning />
+        {warnings.length ? (
+          warningsList(warnings)
+        ) : (
+          <EmptyListMessage
+            mainMessage={`Warnings list is empy`}
```
Comment (anton-vlasik) [inline src/components/Warnings/Warnings.jsx]: empty

File: src/components/Warnings/Warnings.jsx (hunk 1 above)
Comment (glebdronchenko) [reply src/components/Warnings/Warnings.jsx]: Done.

File: src/components/Warnings/Warnings.jsx (hunk 2 above)
Comment (glebdronchenko) [reply src/components/Warnings/Warnings.jsx]: I have removed this magic string and have connected redux to this component.

File: src/components/Warnings/Warnings.jsx (hunk 3 above)
Comment (glebdronchenko) [reply src/components/Warnings/Warnings.jsx]: Done. I have renamed this function.

File: src/components/Warnings/Warnings.jsx (hunk 4 above)
Comment (glebdronchenko) [reply src/components/Warnings/Warnings.jsx]: This props was renamed in this component.

File: src/components/Warnings/Warnings.jsx (hunk 5 above)
Comment (glebdronchenko) [reply src/components/Warnings/Warnings.jsx]: I have corrected this word and have moved this magic string to constants.

File: src/utils/constants.js [hunk 6]
```diff
@@ -10,3 +10,6 @@ export const DAILY_FORECAST_TITLE_TEXT = 'Daily forecast symbol for';
 
 export const CURRENT_CITY_FORECAST_ALT_TEXT = 'Current city forecast';
 export const CURRENT_CITY_FORECAST_TITLE_TEXT = 'Current city forecast';
+
+export const EMPTY_WARNING_LIST_MAIN_MESSAGE = 'Warnings list is empty';
+export const EMPTY_WARNING_LIST_ADDITIONAL_MESSAGE = 'Have a nice day!';
```
Comment (anton-vlasik) [inline src/utils/constants.js]: This is overhead. Either you use i18n and create json vocabularies. Or just use strings as is. They are one of a kind and everybody understands that that is a user text

Comment (glebdronchenko) [reply src/utils/constants.js]: Done. I have moved this values to props of component.

**Review by anton-vlasik — APPROVED** [review]

**Review by ximet — APPROVED** [review]

**Pull request merged**